### DISCUSSION TITLE
keymap to equal components

### DISCUSSION THREAD

**Discussion by hbak** [ldelossa/nvim-ide#91] [Ideas]
How would you feel about a component buffer keymap that equalized the size of all components in the panel?  It would be a sibling to minimize (-) and maximize (+).   I realize one could simply map :Workspace reset, but IMO it would be super convenient and sensible to have a default buffer map (eg. "=") that set all your components back to equal sizing if you've minimized or maximized.

**Comment by ldelossa** [comment]
So, you'd like to hit "=" inside a component to do the equivalent of "Workspace Reset"?

**Reply by hbak** [reply]
Yes :)

**Reply by hbak** [reply]
But, i'm also checking with you to see whether you feel that's anti-canonical or something.

**Reply by ldelossa** [reply]
Sounds fine to me. I dont think "=" is canonically used for anything in any component. Id be fine with having it just call "Workspace Reset" if encountered within a component win.

**Reply by ldelossa** [reply]
this has been implemented here: https://github.com/ldelossa/nvim-ide/commit/892d71e64f587289ba7b868e5acf0c54e4e353ea
